rotkehlchen/history/manager.py

```diff
@@ -159,6 +159,7 @@ def query_location_latest_trades(
         if exchanges_list is None:
             return
 
+        log.debug(f'Querying trades from {exchanges_list=}')
```

Comment:
Why do you need this debug entry? Doesn't also each function call below log queries too?